_editorial_policies/02_reviewing.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: single
+sidebar:
+  nav: reviewing.md
+title: Policies About Review of Manuscripts
+excerpt: A draft of the policies for reviewing manuscripts
+permalink: /policies/reviewing/
+---
+
+## Conflicts of interest
+
+A reviewer may not have conflict of interest with an author of paper
+they are reviewing. If a reviewer identifies a conflict of interest
+when invited to review, they should notify the inviting editor. If a
+conflict of interest arises during the review process, they should
+immediately notify the inviting editor. For the definitions of a
+conflict of interest, see the [definition in the LiveCoMS
+bylaws](/policies/livecoms_bylaws/).
+
+
+Reviewers is to evaluate the quality, relevance, and merit of
```

Comment:
Reviewers are to

Reply:
Also this should not be in COI section and perhaps should not be here but merely give a link out to instructions to reviewers?